wordpress-mobile/WordPress-iOS
# Activity log content routes

A recent change in the backend made the routing of Activity Log actions to break, opening all comment and posts links on WebViews.

Since those changes are aimed to be how routing will start working from now on, and since this feature is not yet public, we decided to make a fast fix on the mobile side.

The changes are:
- Ranges will have a `section` instead of a `type`. This with correlation to the sections where the routing should point to.
- Comments `post_id` was renamed `root_id`. It was originally removed but we need it to show the comments of a post. We took the opportunity to rename it to make it more generic.

@loremattei I would like to merge it to the release branch so we avoid releasing the broken version.
@ctarda The changes are quite small. Most of them are updating the test data.

Also a heads up to @malinajirka and @planarvoid about these changes.

**To tests:**
- Build and run.
- Go to the Activity Log.
- Open a comment event.
- Check that both, the comment and the post open in a native view.
